tests/30rooms/60version_upgrade.pl

```diff
@@ -59,6 +61,43 @@ sub upgrade_room {
    });
 }
 
+=head2 is_direct_room
+
+    is_direct_room( $user, $room_id )->then( sub {
+        my ( $is_direct ) = @_;
+    })
+
+Check if a room ID is considered direct by the given user.
```

Comment:
I found this slightly hard to parse; how about something like:

```suggestion
Check if a room ID is considered to be a direct chat by the given user.
```